app/presenters/formats/place_presenter.rb

```diff
@@ -0,0 +1,46 @@
+module Formats
+  class PlacePresenter < EditionFormatPresenter
+  private
+
+    GOVSPEAK_FIELDS = %i(
+      introduction
+      more_information
+      need_to_know
+    ).freeze
+
+    def schema_name
+      'place'
+    end
+
+    def document_type
+      'place'
+    end
+
+    def details
+      details = { external_related_links: external_related_links }
+
+      details = details.merge(govspeak_fields)
```

Comment:
I've changed it to hopefully make it easier to read, let me know what you think.